django_mongodb/compiler.py

```diff
@@ -32,6 +32,33 @@ def __init__(self, *args, **kwargs):
         # A list of OrderBy objects for this query.
         self.order_by_objs = None
 
+    def _unfold_column(self, col):
+        """
+        Flattens a field by returning its target or by replacing dots with GROUP_SEPARATOR
```

Comment:
[PEP 257](https://peps.python.org/pep-0257/) verb style: Flatten

Reply:
Cool wow, I didn't know about that.